tests/drop_counters/exec_template.yml

```diff
@@ -0,0 +1,14 @@
+# ansible-playbook ../tests/drop_counters/exec_template.yml -i lab -l DUT_HOSTNAME --extra-vars "VAR_EXAMPLE=VALUE VAR_EXAMPLE=VALUE" -vvvvv
```

Comment:
Maybe using paramiko directly will simplify this a lot? What do you think?

Reply:
As we still use ansible from pytest and there is possibility to execute jinja templates from ansible, suach approach was used.